packages/ui/src/containers/CreateAccount/index.tsx

```diff
@@ -155,7 +155,7 @@ const CreateAccount = ({ className }: Props) => {
   useEffect(() => {
     // TODO: username exist error message
     if (error) {
-      setToast(t('error.username_exists'));
+      setToast(error.message);
```

Comment:
Is it possible to use `t(error.message)`? This way it seems we can still leverage the i18n phrases and use the original error message as a fallback